### ISSUE TITLE
es-import-bulk can not support 5.x since search_type=scan is not avialable

### ISSUE THREAD

**Issue by zqben402** [skratchdot/elasticsearch-tools#17]
Hi
I am using es-import-bulk in 5.x but it has some compatibility issue. The search_type=scan is not available anymore in 5.x.

**Comment by jurigis** [comment]
I forked the project and upgraded to the newest elasticsearch-js library and removed the search_type property. I've tested the changes successfully with my local 5.4.1 elasticsearch installation.

**Comment by Sujeet1604** [comment]
"es-export-bulk is not recognised as internal or external command" in windows cmd ...any help?

**Comment by jurigis** [comment]
Did you install it with the -g switch (npm install -g). You could also verify the PATH variable to contain the global install path.

**Comment by zqben402** [comment]
Thanks Jurigis. So I just need to uninstall and install again to apply your changes?

**Comment by jurigis** [comment]
Yes. It should work.

**Comment by zqben402** [comment]
I tried 
npm uninstall -g elasticsearch-tools
and 
npm install -g elasticsearch-tools
But it still throws me the error.

**Comment by jurigis** [comment]
The "npm install" should display the path where the tools are installed. Is this path in your PATH environment variable?

**Comment by zqben402** [comment]
I still seeing searchType: 'scan' in both
https://github.com/skratchdot/elasticsearch-tools
and 
https://github.com/jurigis/elasticsearch-tools in es-export-bulk.js.

Am I cloning the wrong version?

**Comment by zqben402** [comment]
I just manually edit the es-export-bulk.js and now it works !. Thanks

**Comment by zqben402** [comment]
Hi Jurigis:

I can successfully dump the document out with deleting 'scan' search type in the code. However the speed of dumping is like 7~8 times slower than before even with "sort":["_doc"] in the query.

Do you have any idea why this is slow?

**Comment by Sujeet1604** [comment]
Yeah .It worked.. I changed the PATH variable to c:/user/user_name/appdata

**Comment by abelsromero** [comment]
thanx for the fix @jurigis !
Any change this could be merged into master?

**Comment by jurigis** [comment]
I'm not able to merge it into the master branch. Only @skratchdot could do it. I've created a pull request. But there are several pull requests with failed integration tests.
Does anyone know how to fix the integration tests and who decides when to merge them into the master?

**Comment by skratchdot** [comment]
Sorry about this.  @jurigis - I just merged your request.  I don't think there are any tests in this project- but it would be great if there were.  I haven't actually used this since my previous job (where it was necessary). I'll try to take another look at this and clean things up.  Thanks for the PR

**Comment by skratchdot** [comment]
I've published a new version: `elasticsearch-tools@4.0.0`.  Can you see if this is still an issue?  I added a few tests today, but plan on adding more.

**Comment by jurigis** [comment]
Good work! With your new version the issue is solved. I've just tested it with my local elasticsearch installation.
